challenge-react/src/components/Card/Card.styled.jsx

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { pxToVw } from '@/helpers';
+
+export const Card = styled.div`
+  display: flex;
+  border-radius: 4px;
+  box-shadow: 2px 2px 2px 2px #e2e2e2;
+  width: ${pxToVw(320, 320)};
+  min-height: ${pxToVw(200, 320)};
+  flex-direction: column;
+  position: relative;
+  margin: ${pxToVw(20)};
+  height: 100%;
+  @media (min-width: 768px) {
+    width: ${pxToVw(320, 768)};
+    min-height: ${pxToVw(200, 768)};
+    height: 100%;
```

Comment:
I think the height: 100%; is redundant. Because it has the same value as the base class.